public/app/plugins/datasource/cloudwatch/partials/query.parameter.html

```diff
@@ -31,18 +31,31 @@
 	</div>
 </div>
 
-<div class="gf-form-inline">
+<div class="gf-form-inline" ng-if="target.statistics.length === 1">
 	<div class="gf-form">
-		<label class="gf-form-label query-keyword width-8">
+		<label class=" gf-form-label query-keyword width-8 ">Id</label>
+		<input type="text " class="gf-form-input " ng-model="target.id " spellcheck='false' ng-model-onblur ng-change="onChange() ">
+	</div>
+	<div class="gf-form max-width-30 ">
+		<label class="gf-form-label query-keyword width-7 ">Expresion</label>
```

Comment:
Typo here: `expresion`

Reply:
I forgot to fix it. Thanks for pointing out.